packages/sdk/android/src/main/java/com/primerioreactnative/datamodels/ach/AchUserDetailsStepRN.kt

```diff
@@ -0,0 +1,22 @@
+package com.primerioreactnative.datamodels.ach
+
+import kotlinx.serialization.Serializable
+import com.primerioreactnative.datamodels.NamedComponentStep
+import com.primerioreactnative.datamodels.PrimerIssuingBankRN
+
+@Serializable
+internal sealed interface AchUserDetailsStepRN : NamedComponentStep {
+    @Serializable
+    class UserDetailsCollectedRN : AchUserDetailsStepRN {
+        override val stepName: String = "userDetailsCollected"
+    }
+
+    @Serializable
+    data class UserDetailsRetrievedRN(
+        val firstName: String,
+        val lastName: String,
+        val emailAddress: String
+    ) : AchUserDetailsStepRN {
+        override val stepName: String = "userDetailsRetrieved"
+    }
+}
```

Comment:
[nit]: new line, we should enable linters in this repo one day :)

Reply:
Will add new line, noted.